### PR TITLE
Improvements to the multi select functionality for guiders

### PR DESCRIPTION
**Pull request by Guntrisoft**
screen shot 2016-10-06 at 11 57 05

- created a module called guiders-multi-action which
  handles showing the action panel
- added a bunch of pub/sub methods to application.js
- published an event in the multi-checkbox module
  which is then listened for in the guiders-multi-action
  module
- published a 'searchComplete' event in sortable-list
  which is also listened for in guiders-multi-action
- altered breadcrumb for groups page
- added action button for managing tags, and altered
  schedule edit to a calendar glyphicon
- added core.js for polyfill for some es6 methods

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -15,4 +15,21 @@
 //= require fullcalendar
 //= require listjs
 //= require daterangepicker
+//= require core.js/core.js
 //= require_tree .
+
+(function($) {
```
Comment (benbarnett) [inline app/assets/javascripts/application.js]: Not sure how crucial this is, but maybe worth moving this stuff into it's own file? `pubsub.js` or something?

File: app/assets/javascripts/guiders-multi-action.es6 [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    }
+
+    bindEvents() {
+      $.subscribe('multi-checkbox-change', this.handleSelection.bind(this));
+      $.subscribe('list-search-complete', this.handleSearchComplete.bind(this));
+      this.$goButton.on('click', this.handleGoClick.bind(this));
+    }
+
+    handleSearchComplete(event, list) {
+      this.handleSelection(event, ...$(list.list).find('[data-multi-checkbox="item"]'));
```
Comment (benbarnett) [inline app/assets/javascripts/guiders-multi-action.es6]: 👀
⑥
🤘

**Comment by benlovell** [comment]
LGTM. ☘

**Comment by benbarnett** [comment]
Awesome 👍

**Pull request merged**